### ISSUE TITLE
Define weapon damage within ItemGroups.conf

### ISSUE THREAD

**Issue by NeumimTo** [Sponge-RPG-dev/NT-RPG#94]
Define weapon damage within ItemGroups.conf and optionally allow to override the item damage from class conf.

**Comment by hellbringer616** [comment]
This sounds like it would take a lot of effort out of configuring classes.
Would it possible to add the ability to define a WeaponClass within the Classes/Races.conf files?
```
AllowedWeapons: [
     WeaponClass:Swords
     WeaponClass:Maces
]
```

or are you against the idea?

**Comment by NeumimTo** [comment]
solved in 05e0ee7

**Comment by NeumimTo** [comment]
i meant 5f5760d
